tests/unit/test_file_operations.py

```diff
@@ -24,24 +23,24 @@ class TestFileOperations(unittest.TestCase):
     """
 
     def setUp(self):
-        self.test_file = "test_file.txt"
+        self.config = Config()
```

Comment:
If someone knows how to make unittest classes use pytest fixtures, let me know.  Otherwise I think we should add a backlog item to refactor things so we're using pytest only.